### PR TITLE
Suppress missing attribute class errors

### PR DESCRIPTION
**Pull request by othercorey**

### CODE REVIEW

File: src/Command/Command.php [hunk 1]
```diff
@@ -29,7 +30,7 @@
  * Includes traits that integrate logging
  * and ORM models to console commands.
  */
-#[\AllowDynamicProperties]
+#[AllowDynamicProperties]
```
Comment (dereuromark) [inline src/Command/Command.php:33]: Why not keeping fcqn?

Comment (ADmad) [reply src/Command/Command.php:33]: Why does Command still need the annotation in 5.x?

Comment (othercorey) [reply src/Command/Command.php:33]: phpcs throws an errors about fully qualified names.

Comment (dereuromark) [reply src/Command/Command.php:33]: That is weird, we do not have this in other places.

Comment (othercorey) [reply src/Command/Command.php:33]: We have had with each attribute and had to change it when merging into 5.x.

All other fully qualified names were cleaned up.

Comment (othercorey) [reply src/Command/Command.php:33]: > Why does Command still need the annotation in 5.x?

Probably doesn't. Was just trying to fix this constant error we have.

**Comment by othercorey** [comment]
Any objection to the error suppression? If we add any attribute to 5.x, we're going to hit this error.

**Comment by dereuromark** [comment]
> Any objection to the error suppression?

We can always remove it in the future once the issue is solved. So sounds fine.

**Comment by othercorey** [comment]
Would be nice if they ignored built-in attributes in earlier versions of php. The whole argument for the attribute syntax as users can add to code that supports a range of php versions.

**Comment by dereuromark** [comment]
Didnt you just need the polyfill of 8.1 to solve this? But yeah, a bit annoying with supporting multiple PHP versions.

**Comment by dereuromark** [comment]
The original question still stands, though: "Why does Command still need the annotation in 5.x?"
For me the tests ran on both 8.0 and 8.1 just fine. Is this specific to 8.2+?

**Comment by othercorey** [comment]
This is for 8.2. The attribute can probably be dropped.

**Pull request merged**